### DISCUSSION TITLE
default rules for system themes. Let's discuss your expectations

### DISCUSSION THREAD

**Discussion by math-GH** [FreshRSS/FreshRSS#4642] [Show and tell]
Hi,

right now I work on the themes to reduce the general maintenance work load and give simpler options to customize the themes.

What do you expect from system themes?

That is in my mind:

1. variables are stored in `:root { ..... }` (reason 1: to have 1 point of truth; reason 2: it is very easy to customize the colors f.e. with CustomCSS)
2. ALL colors are stored in variables (reason: it gives the full picture of used colors. It reduce the used similar colors)
3. use `rem` instead of `em` (`em` only with a reason)
4. if possible use `0.25rem`, `0.5rem`, `0.75rem`, `1rem`, `1.25rem` etc.
5. (It is my vision for future release: [offer dark themes of light themes](https://github.com/FreshRSS/FreshRSS/labels/FR%3A%20dark%20themes). It needs a lot of improvements)

**Comment by Alkarex** [comment]
Sounds like good improvements 👍🏻 
Bonus if it could be more or less the same variables between themes

**Comment by aledeg** [comment]
That would be great!
